certmgr/mgr/file.go

```diff
@@ -201,6 +215,20 @@ func ReadSpecFile(path string, defaults *ParsableSpecOptions) (*cert.Spec, error
 	// transfer the parsed durations into their final resting spot.
 	spec.FinalizeSpecOptionParsing()
 
+	if len(spec.ParsedKeyUsages) > 0 {
+		for _, KeyUsageRaw := range spec.ParsedKeyUsages {
+			keyUsage, ok := config.ExtKeyUsage[strings.ToLower(KeyUsageRaw)]
+			if !ok {
+				return nil, fmt.Errorf("spec %s specifies unknown key usage '%s'. Valid values are: [%v]", path, KeyUsageRaw, validExtUsage)
+			} else {
+				spec.KeyUsages = append(spec.KeyUsages, keyUsage)
```

Comment:
You can remove the else clause, as you have a return.

Reply:
Fixed, thanks, originally this wasnt a return and I didn't catch this when refactoring.